Mechanisms_evolution/Mechanisms_Evolution.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Lab 2: Mechanisms of Evolution'
+title: 'BIO104 Lab A2: Mechanisms of Evolution'
```

Comment:
Why A2?

Reply:
A is for assignment - may be a relic of the past though

Reply:
use A